src/TurnerSoftware.SitemapTools/Parser/XmlSitemapParser.cs

```diff
@@ -14,18 +15,20 @@ namespace TurnerSoftware.SitemapTools.Parser
 	public class XmlSitemapParser : ISitemapParser
 	{
 #pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
-		public async Task<SitemapFile> ParseSitemapAsync(TextReader reader)
+		public async Task<SitemapFile> ParseSitemapAsync(TextReader reader, CancellationToken cancellationToken = default)
 #pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
 		{
 			var result = new SitemapFile();
 			XDocument document;
 
 			try
 			{
-#if NETSTANDARD2_1
-				document = await XDocument.LoadAsync(reader, LoadOptions.None, default);
+#if (NETSTANDARD2_1 || NETCOREAPP)
```

Comment:
Not sure why there is `NETCOREAPP` added here - the only targets for the project are .NET Standard 2 and 2.1 - was this added accidentally while testing this change or was there a particular issue with `NETSTANDARD2_1`?

Reply:
I removed it.
I added this to try to pass test, but it does not work then I changed test and forgot remove this.
Thanks.